### ISSUE TITLE
Convert ADMG to DAG with boolean attributes

### ISSUE THREAD

**Issue by cthoyt** [y0-causal-inference/y0#37]
Go for it!  Currently, ADMG's represent hidden variables with a bidirected edge.  We should create a function that transforms an ADMG into a DAG where each node has a boolean atttribute called is_latent or is_hidden

**Comment by djinnome** [comment]
An ADMG that is converted to a latent variable DAG, it should convert back to the same ADMG.  But if we select a subset of nodes and toggle their hidden attribute, then apply Robin Evans' algorithm it will result in a different ADMG.

**Comment by cthoyt** [comment]
> An ADMG that is converted to a latent variable DAG, it should convert back to the same ADMG. But if we select a subset of nodes and toggle their hidden attribute, then apply Robin Evans' algorithm it will result in a different ADMG.

Correct. The proposed PR #38 tests that a round trip can be made on an unmodified graph (and effectively that the forward and reverse conversion work)
